buildSrc/src/testKit/testclusters/build.gradle

```diff
@@ -63,10 +63,11 @@ task printLog {
 
 task upToDate1 {
     useCluster testClusters.myTestCluster
-}
-
-task upToDate2 {
-    useCluster testClusters.myTestCluster
+    outputs.upToDateWhen { true }
+    inputs.file(file("build.gradle"))
```

Comment:
I think hard-coding `upToDateWhen` makes declaring this input unnecessary.

Reply:
I think it was needed since the task has no inputs.

Reply:
I tested this locally and it works fine with a task with no inputs.